lib/handler/configurator/mruby.c

```diff
@@ -31,23 +31,29 @@ struct mruby_configurator_t {
     h2o_configurator_t super;
     h2o_mruby_config_vars_t *vars;
     h2o_mruby_config_vars_t _vars_stack[H2O_CONFIGURATOR_NUM_LEVELS + 1];
+    mrb_state *mrb; /* will be lazily initialized */
```

Comment:
configurator also should use one shared mrb_state for compile testing, because handlers may cause side-effects to other handlers, and it may cause compile errors.